cfme/scripting/appliance.py

```diff
@@ -22,6 +22,19 @@ def main():
     pass
 
 
+@click.group(help='Helper commands for db')
```

Comment:
just use 2 lists for 2 command roots 

note that all those db calls should move to a sub-object to begin with, so this will break